Fix broken CI

Rot127 opened the pull request:
# SQUASH ME

commit message:

Fix Hexagon CI tests:
- Increase timeout for emulation tests for OpenBSD
- Use monotonic clock for timestamps (otherwise two packets can get easily the same timestamp).

Some cleanups
- Replace some `calloc` with `RZ_NEW0`
- Add returns after fatal logs.
- Properly reset all members of a `HexPkt` when cleaned.

 <!-- Filling this template is mandatory -->

**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

<!-- Explain the **details** for making this change. Is a new feature implemented? What existing problem does the pull request solve? How does the pull request solve these issues? Please provide enough information so that others can review your pull request. -->


**TODO**

Windows:

```
[**]            C:\projects\rizin\test\db\analysis\java    18776 OK      1001 BR        0 XX       34 FX
[XX] C:\projects\rizin\test\db\analysis\hexagon hexagon last instr. is endloop branch.
ANSICON=1 RZ_NOPLUGINS=1 C:\projects\rizin\rizin-clang_cl_64_dyn-v0.8.0\bin\rizin.exe -escr.utf8=0 -escr.color=0 -escr.interactive=0 -eflirt.sigdb.load.system=false -eflirt.sigdb.load.home=false -N -Qc 'e analysis.jmp.cref=true
aaa
s 0x00005abc
pd 22
' bins/elf/analysis/hexagon-hello-loop
-- stdout
--- expected
+++ actual
@@ --1,11 +-1,11 @@
 |       ,=< 0x00005abc      ?       if (P3) jump:nt 0x5b1c
-|       |   ; CODE XREF from sym.memcpy @ 0x5abc
+|       |   ; CODE XREF from fcn.00005a90 @ 0x5abc
 |      ,==< 0x00005ac0      /       loop0(0x5acc,R4)
 |      ||   0x00005ac4      |       P0 = cmp.gtu(R4,##0x1)
 |      ||   0x00005ac8      \       R8 = R4
-|      ||   ; CODE XREFS from sym.memcpy @ 0x5ac0, 0x5b10
+|      ||   ; CODE XREFS from fcn.00005a90 @ 0x5ac0, 0x5b10
 |     .`--> 0x00005acc      /       if (P0) R17 = add(R17,##0x20)
 |     : |   0x00005ad0      \       dcfetch(R5+#0x0)
 |     : |   0x00005ad4      /       P3 = cmp.eq(R4,R8)
-- stderr
ERROR: Cannot determine entrypoint, using 0x00005000.
[x] Analyze all flags starting with sym. and entry0 (aa)
[x] Analyze function calls
[x] find and analyze function preludes
[x] Analyze len bytes of instructions for references
[x] Check for classes
[x] Finding xrefs in noncode section with analysis.in=io.maps
[x] Analyze value pointers (aav)
[x] Value from 0x00000000 to 0x00003ec0 (aav)
[x] 0x00000000-0x00003ec0 in 0x0-0x3ec0 (aav)
[x] 0x00000000-0x00003ec0 in 0x4000-0xb764 (aav)
[x] Value from 0x00004000 to 0x0000b764 (aav)
[x] 0x00004000-0x0000b764 in 0x0-0x3ec0 (aav)
[x] 0x00004000-0x0000b764 in 0x4000-0xb764 (aav)
[x] Emulate functions to find computed references
[x] Analyze local variables and arguments
[x] Applied 0 FLIRT signatures via sigdb
[x] Propagate noreturn information
[x] Integrate dwarf function information.
[x] Resolve pointers to data sections
[x] Use -AA or aaaa to perform additional experimental analysis.
[XX] C:\projects\rizin\test\db\analysis\hexagon hexagon basic block graph
ANSICON=1 RZ_NOPLUGINS=1 C:\projects\rizin\rizin-clang_cl_64_dyn-v0.8.0\bin\rizin.exe -escr.utf8=0 -escr.color=0 -escr.interactive=0 -eflirt.sigdb.load.system=false -eflirt.sigdb.load.home=false -N -Qc 'aaa
s main
agf
' bins/elf/analysis/hexagon-hello-loop
-- stdout
--- expected
+++ actual
@@ -29,7 +29,7 @@
 |    |                                         |
 |.------------------------------------.    .-----------------.
 ||  0x5154                            |    |  0x5134         |
-|| ; DATA XREF from main @ 0x5130     |    | [   jump 0x5138 |
+|| ; DATA XREF from main @ 0x5130     |    | ?   jump 0x5138 |
 || ?   R0 = ##0x0                     |    `-----------------'
 || [   LR:FP = dealloc_return(FP):raw |        v
 |`------------------------------------'        |
-- stderr
ERROR: Cannot determine entrypoint, using 0x00005000.
[x] Analyze all flags starting with sym. and entry0 (aa)
[x] Analyze function calls
[x] find and analyze function preludes
[x] Analyze len bytes of instructions for references
[x] Check for classes
[x] Finding xrefs in noncode section with analysis.in=io.maps
[x] Analyze value pointers (aav)
[x] Value from 0x00000000 to 0x00003ec0 (aav)
[x] 0x00000000-0x00003ec0 in 0x0-0x3ec0 (aav)
[x] 0x00000000-0x00003ec0 in 0x4000-0xb764 (aav)
[x] Value from 0x00004000 to 0x0000b764 (aav)
[x] 0x00004000-0x0000b764 in 0x0-0x3ec0 (aav)
[x] 0x00004000-0x0000b764 in 0x4000-0xb764 (aav)
[x] Emulate functions to find computed references
[x] Analyze local variables and arguments
[x] Applied 0 FLIRT signatures via sigdb
[x] Propagate noreturn information
[x] Integrate dwarf function information.
[x] Resolve pointers to data sections
[x] Use -AA or aaaa to perform additional experimental analysis.
[**]           C:\projects\rizin\test\db\analysis\h8300    18777 OK      1001 BR        0 XX       34 FX

```

**Test plan**

<!-- What steps should the reviewer take to test your pull request? Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots/videos. This is your time to re-check that everything works and that you covered all the edge cases -->

All green

**Closing issues**

<!-- put "closes #XXXX" in your comment to auto-close the issue that your PR fixes (if any). -->

...

Rot127 committed:
Increase timeout for OpenBSD

XVilka commented:
This error sometimes happens, sometimes - not. Probably some uninitialized data?
```
[XX] C:\projects\rizin\test\db\analysis\hexagon hexagon basic block graph
ANSICON=1 RZ_NOPLUGINS=1 C:\projects\rizin\rizin-vs2022_64-v0.8.0\bin\rizin.exe -escr.utf8=0 -escr.color=0 -escr.interactive=0 -eflirt.sigdb.load.system=false -eflirt.sigdb.load.home=false -N -Qc 'aaa
s main
agf
' bins/elf/analysis/hexagon-hello-loop
-- stdout
--- expected
+++ actual
@@ -29,7 +29,7 @@
 |    |                                         |
 |.------------------------------------.    .-----------------.
 ||  0x5154                            |    |  0x5134         |
-|| ; DATA XREF from main @ 0x5130     |    | [   jump 0x5138 |
+|| ; DATA XREF from main @ 0x5130     |    | ?   jump 0x5138 |
 || ?   R0 = ##0x0                     |    `-----------------'
 || [   LR:FP = dealloc_return(FP):raw |        v
 |`------------------------------------'        |
-- stderr
ERROR: Cannot determine entrypoint, using 0x00005000.
[x] Analyze all flags starting with sym. and entry0 (aa)
[x] Analyze function c
```

Rot127 committed:
Reset all members of a packet.

Rot127 committed:
Replace calloc with Rizin macros

Rot127 committed:
Add returns after fatal logs

XVilka reviewed (approved):
LGTM if green

XVilka commented:
Didn't help for `clang-cl` build though:
```
[XX] C:\projects\rizin\test\db\analysis\hexagon hexagon basic block graph
ANSICON=1 RZ_NOPLUGINS=1 C:\projects\rizin\rizin-clang_cl_64_dyn-v0.8.0\bin\rizin.exe -escr.utf8=0 -escr.color=0 -escr.interactive=0 -eflirt.sigdb.load.system=false -eflirt.sigdb.load.home=false -N -Qc 'aaa
s main
agf
' bins/elf/analysis/hexagon-hello-loop
-- stdout
--- expected
+++ actual
@@ -29,7 +29,7 @@
 |    |                                         |
 |.------------------------------------.    .-----------------.
 ||  0x5154                            |    |  0x5134         |
-|| ; DATA XREF from main @ 0x5130     |    | [   jump 0x5138 |
+|| ; DATA XREF from main @ 0x5130     |    | ?   jump 0x5138 |
 || ?   R0 = ##0x0                     |    `-----------------'
 || [   LR:FP = dealloc_return(FP):raw |        v
 |`------------------------------------'        |
-- stderr
ERROR: Cannot determine entrypoint, using 0x00005000.
[x] Analyze all flag
```

Rot127 commented:
Yeah, it is something different. Will investigate tomorrow further.

Rot127 commented:
Yeah, it is something different. At some point the execution paths diverge. Don't know yet why. Will investigate tomorrow further.

Rot127 committed:
Init time structs with 0.

Rot127 committed:
Use monotonic clock to set last access timestamps.

Because instruction packets get pushed out of the buffer based in their last access,
using the real-time clock leads to problems. On Windows the real time clock seems to
not always give ascending values with multiple calls
(real-time clock = best bet of machine of current day time, which can fluctuate).

Using the monotonic clock will always give strictly ascending timestamps.

Due to the use of the real-clock, the instruction packets got
sometimes pushed out of the buffer non-deterministically.

Rot127 commented:
Let's hope [the last commit](https://github.com/rizinorg/rizin/pull/4376/commits/c87c6a52db9cfff76c0d35d7e8c517bc7f91a33e) will actually fix it.
